code/AssetLib/Blender/BlenderLoader.h

```diff
@@ -180,10 +180,17 @@ class BlenderImporter : public BaseImporter, public LogFunctions<BlenderImporter
             const Blender::MTex *tex,
             Blender::ConversionData &conv_data);
 
-    // Returns an error if it can't parse the magic token.
-    std::string ParseMagicToken(
-            std::shared_ptr<IOStream>& stream,
-            std::vector<char>& uncompressed) const;
+    struct StreamOrError {
```

Comment:
Add a comment that this should use a `std::variant` when we move to C++17.

Reply:
Done.